diagnostic.md

```diff
@@ -45,7 +45,11 @@ while (x < 5) {
 
 How many times does the loop run?  What's the value of `x` after the while loop?
 
-_Replace this text with your answer._
+1. x = 1
+2. x = 2
+3. x = 4
+
+The loop runs 3 times.
```

Comment:
You are correct about the number of iterations that the loop runs.  However, x's final value is 8 once the loop terminates.  After the first iteration, x = 1 + 1 = 2.  After the second iteration, x = 2 + 2 = 4.   Finally, the third iteration yields x = 4 + 4 = 8.